src/electron/find_tap_device_name.bat

```diff
@@ -0,0 +1,80 @@
+:: Copyright 2020 The Outline Authors
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::      http://www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+
+:: Queries the registry in order to retrieve the most recently installed TAP network adapter's name.
+:: Accepts a single argument, the name of an environment variable to store the adapter's name.
+:: Exits with a non-zero exit code on failure.
+:: Usage example: find_tap_name.bat TAP_NAME
+
+@echo off
+setlocal enabledelayedexpansion
+
```

Comment:
Some documentation comment here would be nice

Reply:
I feel like this is standard batch programming knowledge (think bash shebang). It's also not trivial to explain, so I'd rather have someone look it up than leaving a long comment. Does that make sense?

Reply:
A comment with a link to the relevant documentation seems like a good balance.

I would also suggest `set NET_ADAPTER_CLASS_GUID={4D36E972-E325-11CE-BFC1-08002BE10318}`.

Reply:
Fair enough -done.